tests/Unit/CrudPanel/CrudPanelFakeColumnsTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Backpack\CRUD\Tests\Unit\CrudPanel;
+
+use Backpack\CRUD\Tests\Unit\Models\Article;
+
+class CrudPanelFakeColumnsTest extends BaseDBCrudPanelTest
+{
+    private $emptyFakeColumnsArray = ['extras'];
+
+    private $fakeFieldsArray = [
+        [
+            'name' => 'field',
+            'label' => 'Normal Field',
```

Comment:
I wonder if it is a valid test to see what the world happens if we don't set a label at all seeing as that's actually optional.

Reply:
There are a number of tests doing just that with other input data sets.

Reply:
Ok.